src/list.js

```diff
@@ -0,0 +1,19 @@
+import { getTodos } from './task.js';
+
```

Comment:
- [ ] It would be better to add instructions about how to clone and run the app locally in the README so that non-developers can use the application. Also, it would be nice to have a deployed version of the app so we can view it on the go